tests/test_domain_std.py

```diff
@@ -328,7 +328,7 @@ def test_cmdoption_for_None(app):
     text = (".. program:: ls\n"
             ".. program:: None\n"
             "\n"
-            ".. option:: -l\n")
+            ".. option:: -l=foo|bar\n")
```

Comment:
Could you add a new testcase for this change?
The purpose of this testcase is checking the behavior of option directive when program is None. It's better to check single behavior by one testcase.